config/routes.rb

```diff
@@ -255,8 +255,6 @@
 
     get '/authentication_methods_setup' => 'users/two_factor_authentication_setup#index'
     patch '/authentication_methods_setup' => 'users/two_factor_authentication_setup#create'
-    get '/second_mfa_setup' => 'users/mfa_selection#index'
-    patch '/second_mfa_setup' => 'users/mfa_selection#update'
```

Comment:
Confirmed in production logs from today that these should be safe to remove, there does not appear to be legitimate traffic to these paths anymore.